plugins/plugin-docker/che-plugin-openshift-client/src/test/java/org/eclipse/che/plugin/openshift/client/OpenShiftConnectorTest.java

```diff
@@ -43,6 +46,7 @@
     private static final String   OPENSHIFT_DEFAULT_USER_PASSWORD = "devel";
     private static final int      OPENSHIFT_LIVENESS_PROBE_DELAY = 300;
     private static final int      OPENSHIFT_LIVENESS_PROBE_TIMEOUT = 1;
+    private static final String   OPENSHIFT_DEFAULT_TOKEN = "91XMfu-FuNDkGjcIh6b0y1EtCvztGeSsSqRrWhBfyL8";
```

Comment:
@skabashnyuk that's a test class and that's a sample token. What is exactly your security concern?